delivery_postlogistics/views/stock.xml

```diff
@@ -6,6 +6,16 @@
         <field name="model">stock.picking</field>
         <field name="inherit_id" ref="delivery.view_picking_withcarrier_out_form" />
         <field name="arch" type="xml">
+            <field name="state" position="before">
+                <button
+                    name="action_generate_carrier_label"
+                    help="Create Shipping Label 🚚"
+                    attrs="{'invisible': [('state', '!=', 'done')]}"
+                    string="Shipping Label 🚚"
```

Comment:
```suggestion
                    string="Create Shipping Label 🚚"
```

Reply:
hello @nilshamerlinck, this PR has been updated.